Capri.Services/IUserGetter.cs

```diff
@@ -0,0 +1,13 @@
+﻿using Capri.Database.Entities.Identity;
```

Comment:
That file should be in some other directory like `ApplicationUser`.  Also it will be nice to rename it to `IApplicationUserGetter` to remove ambigous with `IUserGetter`